src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -251,154 +397,122 @@ export class ScriptEvaluator {
             executionContextId: realm.executionContextId,
           }
         );
-        // TODO (#375): release `argEvalResult.result.objectId` after using.
+        // TODO(#375): Release `argEvalResult.result.objectId` after using.
         return {objectId: argEvalResult.result.objectId};
       }
 
-      // TODO(#375): dispose of nested objects.
+      case 'channel': {
+        const channelName = argumentValue.value.channel;
 
-      default:
-        throw new Error(
-          `Value ${JSON.stringify(argumentValue)} is not deserializable.`
+        const argEvalResult = await realm.cdpClient.sendCommand(
+          'Runtime.callFunctionOn',
+          {
+            functionDeclaration: String(() => {
+              const queue: string[] = [];
```

Comment:
As discussed offline, there can be any value
```suggestion
              const queue: any[] = [];
```